consensus/state.go

```diff
@@ -142,7 +143,7 @@ type State struct {
 	// for reporting metrics
 	metrics *Metrics
 
-	eventCollector *trace.Client
+	traceClient *trace.Client
```

Comment:
apologies for not simply including the name change in a separate PR, it was driving me crazy enough that I just changed it here